cluster.yaml.example

```diff
@@ -148,3 +148,4 @@ apiserverLivenessPath: livez?exclude=etcd
 controlPlaneOperatorImage: registry.svc.ci.openshift.org/ibm-roks/ibm-roks-4.4:control-plane-operator
 controlPlaneOperatorSecurity: 1001
 masterPriorityClass: ''
+roksMetricsImage: registry.svc.ci.openshift.org/hypershift-toolkit/ibm-roks-4.6:roks-metrics
```

Comment:
I assume we're going to have to coordinate some work on our side to ensure we provide this image. I don't see any logic to skip the metrics setup if this isn't provided.